<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewData</class>
 <widget class="QWidget" name="ViewData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Марка</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Mark">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TypeMaterial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип сырья</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAddRow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDeleteRow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>441</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonUploadData">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Загрузить
новые 
данные
из вне</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPasteRow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Текущая дата:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
